BUG: `Urgent` - All PR's are getting deployment errors in git pipeline

### Pandas version checks

- [x] I have checked that this issue has not already been reported.

- [x] I have confirmed this bug exists on the [latest version](https://pandas.pydata.org/docs/whatsnew/index.html) of pandas.

- [x] I have confirmed this bug exists on the [main branch](https://pandas.pydata.org/docs/dev/getting_started/install.html#installing-the-development-version-of-pandas) of pandas.


### Reproducible Example

```python
na
```

### Issue Description

FAILED pandas/tests/io/test_sql.py::test_xsqlite_execute_fail - pandas.errors.DatabaseError: Execution failed on sql 'INSERT INTO test VALUES("foo", "bar", 1.234)': no such column: "foo" - should this be a string literal in single-quotes?
FAILED pandas/tests/io/test_sql.py::test_xsqlite_execute_closed_connection - pandas.errors.DatabaseError: Execution failed on sql 'INSERT INTO test VALUES("foo", "bar", 1.234)': no such column: "foo" - should this be a string literal in single-quotes?
= 2 failed, 3760 passed, 372 skipped, 232245 deselected, 376 xfailed, 1 xpassed, 81 warnings in 1182.58s (0:19:42) =
/home/runner/micromamba/envs/test/lib/python3.12/site-packages/s3fs/core.py:569: DeprecationWarning: There is no current event loop
  loop = asyncio.get_event_loop()
Error: Process completed with exit code 1.
Run actions/upload-artifact@v4
With the provided path, there will be 1 file uploaded
Artifact name is valid!
Root directory input is valid!
Beginning upload of artifact content to blob storage
Uploaded bytes 39831
Finished uploading artifact content to blob storage!
SHA256 hash of uploaded artifact zip is f1126cc96347284a152f921e8e59bcadd2762e4d3872d66fe1d9881a9775948f
Finalizing artifact upload
Artifact Test results.zip successfully finalized. Artifact ID 2636217021
Artifact Test results has been successfully uploaded! Final size is 39831 bytes. Artifact ID is 2636217021
Artifact download URL: https://github.com/pandas-dev/pandas/actions/runs/13478588009/artifacts/2636217021
Run codecov/codecov-action@v4
eventName: pull_request
baseRef: pandas-dev:main | headRef: Anurag-Varma:bug#60723
==> Fork detected, tokenless uploading used
==> linux OS detected
https://cli.codecov.io/latest/linux/codecov.SHA256SUM
gpg: directory '/home/runner/.gnupg' created
gpg: keybox '/home/runner/.gnupg/pubring.kbx' created
gpg: /home/runner/.gnupg/trustdb.gpg: trustdb created
gpg: key 806BB28AED779869: public key "Codecov Uploader (Codecov Uploader Verification Key) <security@codecov.io>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: Signature made Fri Feb 21 21:18:38 2025 UTC
gpg:                using RSA key 27034E7FDB850E0BBC2C62FF806BB28AED779869
gpg: Good signature from "Codecov Uploader (Codecov Uploader Verification Key) <security@codecov.io>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: 2703 4E7F DB85 0E0B BC2C  62FF 806B B28A ED77 9869
==> Uploader SHASUM verified (ec031bdc38a2ca279ea1f56650b63aa6ce1365e7ee9da33b[413](https://github.com/pandas-dev/pandas/actions/runs/13477733861/job/37658956940#step:9:417)e6f175a94fe2b  codecov)
==> Running version latest
Could not pull latest version information: SyntaxError: Unexpected token '<', "<!DOCTYPE "... is not valid JSON
==> Running git config --global --add safe.directory /home/runner/work/pandas/pandas
/usr/bin/git config --global --add safe.directory /home/runner/work/pandas/pandas
==> Running command '/home/runner/work/_actions/codecov/codecov-action/v4/dist/codecov create-commit'
/home/runner/work/_actions/codecov/codecov-action/v4/dist/codecov create-commit --git-service github -B Anurag-Varma:bug#60723 -C 42c51ec8e4efc3c8a221625c92b22e2730b5fe7e
info - 2025-02-23 02:40:28,834 -- ci service found: github-actions
info - 2025-02-23 02:40:28,861 -- Creating a commit for an unprotected branch: Anurag-Varma:bug#60723
info - 2025-02-23 02:40:29,365 -- Process Commit creating complete
==> Running command '/home/runner/work/_actions/codecov/codecov-action/v4/dist/codecov create-report'
/home/runner/work/_actions/codecov/codecov-action/v4/dist/codecov create-report --git-service github -C 42c51ec8e4efc3c8a221625c92b22e2730b5fe7e
info - 2025-02-23 02:40:30,161 -- ci service found: github-actions
info - 2025-02-23 02:40:30,587 -- Process Report creating complete
info - 2025-02-23 02:40:30,588 -- Finished creating report successfully --- {"response": "{\"status\":\"queued\"}\n"}
==> Running command '/home/runner/work/_actions/codecov/codecov-action/v4/dist/codecov do-upload'
/home/runner/work/_actions/codecov/codecov-action/v4/dist/codecov do-upload -F unittests --git-service github -n codecov-pandas -C 42c51ec8e4efc3c8a221625c92b22e2730b5fe7e
info - 2025-02-23 02:40:31,391 -- ci service found: github-actions
warning - 2025-02-23 02:40:31,[418](https://github.com/pandas-dev/pandas/actions/runs/13477733861/job/37658956940#step:9:422) -- xcrun is not installed or can't be found.
warning - 2025-02-23 02:40:31,450 -- No gcov data found.
warning - 2025-02-23 02:40:31,450 -- coverage.py is not installed or can't be found.
info - 2025-02-23 02:40:31,498 -- Found 1 coverage files to report
info - 2025-02-23 02:40:31,498 -- > /home/runner/work/pandas/pandas/coverage.xml
info - 2025-02-23 02:40:32,005 -- Your upload is now processing. When finished, results will be available at: https://app.codecov.io/github/pandas-dev/pandas/commit/42c51ec8e4efc3c8a221625c92b22e2730b5fe7e
info - 2025-02-23 02:40:32,331 -- Process Upload complete

### Expected Behavior

na

### Installed Versions

<details>

Replace this line with the output of pd.show_versions()

</details>

--- a/pandas/tests/io/test_sql.py
+++ b/pandas/tests/io/test_sql.py
@@ -4282,11 +4282,11 @@ def test_xsqlite_execute_fail(sqlite_buildin):
     cur.execute(create_sql)
 
     with sql.pandasSQL_builder(sqlite_buildin) as pandas_sql:
-        pandas_sql.execute('INSERT INTO test VALUES("foo", "bar", 1.234)')
-        pandas_sql.execute('INSERT INTO test VALUES("foo", "baz", 2.567)')
+        pandas_sql.execute("INSERT INTO test VALUES('foo', 'bar', 1.234)")
+        pandas_sql.execute("INSERT INTO test VALUES('foo', 'baz', 2.567)")
 
         with pytest.raises(sql.DatabaseError, match="Execution failed on sql"):
-            pandas_sql.execute('INSERT INTO test VALUES("foo", "bar", 7)')
+            pandas_sql.execute("INSERT INTO test VALUES('foo', 'bar', 7)")
 
 
 def test_xsqlite_execute_closed_connection():
@@ -4304,7 +4304,7 @@ def test_xsqlite_execute_closed_connection():
         cur.execute(create_sql)
 
         with sql.pandasSQL_builder(conn) as pandas_sql:
-            pandas_sql.execute('INSERT INTO test VALUES("foo", "bar", 1.234)')
+            pandas_sql.execute("INSERT INTO test VALUES('foo', 'bar', 1.234)")
 
     msg = "Cannot operate on a closed database."
     with pytest.raises(sqlite3.ProgrammingError, match=msg):

